Sources/Lifecycle/Lifecycle.swift

```diff
@@ -180,9 +180,12 @@ public struct LifecycleShutdownHandler<State> {
 // MARK: - ServiceLifecycle
 
 /// `ServiceLifecycle` provides a basic mechanism to cleanly startup and shutdown the application, freeing resources in order before exiting.
-///  By default, also install shutdown hooks based on `Signal` and backtraces.
+/// By default, also install shutdown hooks based on `Signal` and backtraces.
```

Comment:
```suggestion
///
/// By default, also install shutdown hooks based on `Signal` and backtraces.
```

give it a bit more breathing space from the first sentence